service/command.go

```diff
@@ -16,12 +16,59 @@ package service // import "go.opentelemetry.io/collector/service"
 
 import (
 	"errors"
+	"fmt"
+
+	"go.opentelemetry.io/collector/component"
 
 	"github.com/spf13/cobra"
 
+	"gopkg.in/yaml.v3"
+
+	"go.opentelemetry.io/collector/config"
 	"go.opentelemetry.io/collector/featuregate"
```

Comment:
order of imports is wrong.

Reply:
Thanks. I would check out what the correct order should be